views/term_table.erb

```diff
@@ -70,25 +70,25 @@
             <div class="container">
                 <div class="grid-list grid-list--vertically-center grid-list--narrow-columns">
                     <div class="data-completeness">
-                        <span class="section-toggle section-toggle--selected" data-section-toggle="bio">
+                        <span class="section-toggle section-toggle--selected" data-section-toggle="bio" data-ga-track-click="Selection toggle: bio">
```

Comment:
Wherever you’ve used `Selection` in the `data-section-toggle` attributes here, I think you mean `section`, since the toggles toggle "sections" of the card, not "selections".